missas/core/tests/test_migration_0030_migrate_schedule_location_to_location_table.py

```
@@ -0,0 +1,85 @@
+    assert created_location.slug == "main-church"
+    assert created_location.schedules.count() == 2
+    assertQuerySetEqual(created_location.schedules.all(), schedules, ordered=False)
+
+
+@pytest.mark.django_db
+def test_multiple_parishes_in_the_same_city_with_the_same_location_name(migrator):
+    old_state = migrator.apply_initial_migration(Migration.dependencies[0])
+    OldSchedule = old_state.apps.get_model(app_name, "Schedule")
+    location_name = "Main Church"
+    schedule1 = baker.make(OldSchedule, location=location_name, parish__name="Parish 1")
+    schedule2 = baker.make(OldSchedule, location=location_name, parish__city=schedule1.parish.city, parish__name="Parish 2")
```

Ruff (0.8.2)

Local variable `schedule2` is assigned to but never used

Remove assignment to unused variable `schedule2`

(F841)